src/tools/sdk/_windows.md

```diff
@@ -14,10 +20,33 @@ To install the Dart SDK:
 C:\> choco install dart-sdk
 ```
 
-To **upgrade** the Dart SDK:
+To upgrade the Dart SDK:
 
 ```terminal
 C:\> choco upgrade dart-sdk
 ```
 
+Default installation directory:
+
+```terminal
+C:\tools\dart-sdk
+```
+
+How to change installation directory?
+
+This package uses the help Get-BinRoot, which means that 
+you can set an environment variable called ChocolateyToolsLocation, which 
+controls where this package is extracted to. 
+For more information check out [get-toolslocation].
```

Comment:
Okay, I gave this section some thought and perhaps we should use more of a prose structure to describe these pieces while also avoiding some of the technical details about Chocolatey that may confuse readers. How about something like this, assuming my understanding of the `ChocolateyToolsLocation` environment variable is correct.

```suggestion
By default, the SDK is installed at `C:\tools\dart-sdk`.

To change the default installation location:

Set the `ChocolateyToolsLocation` environment variable
to the desired extraction location.
To read more about how the installer uses this configuration, 
see [Chocolatey's Get-ToolsLocation reference][get-toolslocation].
```

Reply:
Yes that's correct! Definitely looks more clear now.